### ISSUE TITLE
Can't see list of new users created using API

### ISSUE THREAD

**Issue by RaymondNg0054** [heartexlabs/label-studio#2731]
**Describe the bug**

When trying to create a user using the API, the user does not show up in the organization for the Label Studio UI or when I try to show the list of users in the API. 
https://labelstud.io/api#operation/api_users_create
**To Reproduce**
1. curl -X POST https://URL/api/users/ -H 'Authorization: Token TOKEN' -H "Content-Type: application/json" -d '{"username":"test11","email":"test11@testing.com","active_organization":"1"}'

**Expected behavior**
When I create a new user, I expect to see the new user when calling the list of users in the API. 
curl -X GET https;//URL/api/users/ -H 'Authorization: Token TOKEN>'

**Screenshots**

<img width="818" alt="Screen Shot 2022-07-25 at 12 44 55 PM" src="https://user-images.githubusercontent.com/27940173/180862781-1b8c69a1-ca5e-4089-8c60-7253706d715e.png">

**Environment (please complete the following information):**
 - OS: MacOS
 - Label Studio Version 1.4.1post1

**Additional context**
When I try to create the user with the same email address and same username, I get the message that there is a user that is already created with that email.
When I try to create another user with another email and another username, the user ID integer is incremented by 1, but it still does not show when I try to call the list of users in the API.

**Comment by makseq** [comment]
Could you try this one:

```
curl -X POST https://url/api/users/ -H 'Authorization: Token TOKEN' -H "Content-Type: application/json" -d '{"username":"test11","email":"[test11@testing.com](mailto:test11@testing.com)","active_organization":1}'
```
?

**Comment by RaymondNg0054** [comment]
No luck, still experiencing the same issue.


<img width="808" alt="Screen Shot 2022-07-25 at 10 14 17 PM" src="https://user-images.githubusercontent.com/27940173/180929004-9aa06983-87ae-48b0-aa5f-39df35fbe376.png">
.

**Comment by triklozoid** [comment]
@RaymondNg0054 can you try to create user with label-studio from this branch? https://github.com/heartexlabs/label-studio/pull/2801

**Comment by kw90** [comment]
I experienced the same issue and fixed it with #2801. However, how can I set a password for the newly created user?

I'm looking for a way to programmatically add 2 users (besides the default admin user) over an ansible script. Is there any other way than to go through the Label Studio API and the `/api/users/` endpoint?

**Comment by makseq** [comment]
Closed as solved in #2801.

**Comment by makseq** [comment]
You can reset user passwords using this docs: https://labelstud.io/guide/signup.html#Reset-password
However I'm not sure it's available via API.
